platform/clickhouse/connection.go

```diff
@@ -6,9 +6,7 @@ import (
 	"crypto/tls"
```

Comment:
I would suggest to create a new directory `apache_doris` and implement below functionality there. Then proper `initDBConnection` can be invoked here https://github.com/QuesmaOrg/quesma/blob/main/cmd/main.go#L87